### DISCUSSION TITLE
Temperature sensor

### DISCUSSION THREAD

**Discussion by maaalek** [andrei-tatar/node-red-contrib-smartnora#249] [Q&A]
I have a problem with the temperature/humidity sensor. I send him the correct payload

{
   "payload": {
     "temperature": 22,
"humidity": 10
   }
}
Via UI https://smart-nora.eu/my-nora I can see my sensor and the correct values next to it. However, on Google Home (Android) I only see the humidity value, it doesn't show the temperature. If I switch the sensor to report temperature only, nothing is shown in google home and after selecting the device it shows only its properties - do you have any ideas.

**Comment by maaalek** [comment]
via web UI
![image](https://user-images.githubusercontent.com/5096881/222801399-67756036-3f8e-435f-9816-82b4af5f660c.png)
via google home
![image](https://user-images.githubusercontent.com/5096881/222801631-9594115f-cb7d-49c5-af40-8efa7bafda95.png)

**Reply by andrei-tatar** [reply]
@maaalek it's just how Google Home works. If it would've shown the values I wouldn't have added the NORA dashboard, since it would be redundant. 
Btw, until pretty recently, google home was not even showing the humidity...

**Reply by maaalek** [reply]
@andrei-tatar  It's so weird that I don't even know what to write about it. There is any another way to show text (string) in google home via smart nora ?

**Reply by andrei-tatar** [reply]
No, there is no way. It's all under Google's control and Google focuses more on voice integration.

**Reply by maaalek** [reply]
Thx @andrei-tatar . After all, everyone likes to talk to google at 3am. Case closed.
